### ISSUE TITLE
Simple Tower Autosizing 25.6C inlet WB does not comply with 90.1-G

### ISSUE THREAD

**Issue by ecoeficiente** [NREL/EnergyPlus#5020]
Ashrae 90.1-2007/2010 Appendix G states that _condenser water design supply temperature shall be 85F or 10F approaching deign wet-bulb temperature, **whichever is lower**._
That is not possible with the hardcoded Autosize inlet temperature of 25.6C in EnergyPlus.

**Comment by mjwitte** [comment]
Same issue also raised in helpdesk ticket 11215.

**Comment by Nigusse** [comment]
Reviewed the idd and sizing code of existing cooling towers. Only single and two speed simple cooling towers and variable speed MerkelTower need change.

Proposed new input fields to be added:
- Design Inlet Air Dry-Bulb Temperature (TAirInletDB), default 35.0 C (95 F)
- Design Inlet Air Wet-Bulb Temperature (TAirInletWB), default 25.6 C (78 F)
- Design Approach Temperature (dTAppr), default 3.9 C (7 F)
- Design Range Temperature (dTRange), default 5.56 C ( 10 F)

Tower water inlet and outlet temperatures will be derived as follows:
 TwOutlet = TAirInletWB + dTAppr, default 29.44 C (85 F)
 TwInlet = TwOutlet + dTRange, default 35.0 C (95 F) 

Idd Changes will be made to the following three cooling tower objects:
- CoolingTower:SingleSpeed
- CoolingTower:TwoSpeed
- CoolingTower:VariableSpeed:Merkel

Looking at the code, water inlet and outlet temperatures used for tower load and properties (Cp and Density) calculation are different depending on the cooling tower and UA-calculation method selected. Most places use use values from PlantSizingData objects and in other places use a hard wire values.

I am proposing to replace the "PlantSizingData.ExitTemp" and  "PlantSizingData.DeltaT" values used in the cooling tower sizing calculations by new user input default values. Is there any objection?

@mjwitte @Myoldmopar Any objection to this proposed change or alternate suggestion?

**Comment by mjwitte** [comment]
@Nigusse We have a similar issue with coil design inputs.  The user will expect the Sizing:Plant inputs (PlantSizingData.ExitTemp and .DeltaT) to flow into the sizing calcs unless they are overridden by user inputs in the component object.  One option is to have the Sizing:Plant inputs set the defaults for water temps in the tower.  The range and the deltaT are easy, but do you mess with the default for Approach temp to match the exit temp? At a minimum, there should be warnings if the Sizing:Plant inputs are not consistent with the inputs in the tower object.

**Comment by Nigusse** [comment]
OK. If there is no plant sizing object, then use the inputs in the tower object. On the other hand, if there is plant sizing object, the plant sizing object inputs will have precedence. And inputs inconsistencies between plant sizing object and tower inputs will be checked and warning will be provided as needed. I will report the actual approach value used for sizing (calculated from plant exit and wet-bulb temperature) and the user specified approach value in the tower object as a warning information.

**Comment by mjwitte** [comment]
@Nigusse I was suggesting that the inputs in the tower object take precedence if present, but if they are omitted or defaulted, then the plant sizing inputs would take over.  I think that's what happens in Coil:Cooling:Water with some of the autosizable inputs:
```
Coil:Cooling:Water,
  N3 , \field Design Inlet Water Temperature
       \units C
       \autosizable
       \default autosize
       \minimum> 0
  N4 , \field Design Inlet Air Temperature
       \units C
       \autosizable
       \default autosize
       \minimum> 0
  N5 , \field Design Outlet Air Temperature
       \units C
       \autosizable
       \default autosize
       \minimum> 0
  N6 , \field Design Inlet Air Humidity Ratio
       \units kgWater/kgDryAir
       \autosizable
       \default autosize
       \minimum 0
  N7 , \field Design Outlet Air Humidity Ratio
       \units kgWater/kgDryAir
       \autosizable
       \default autosize
       \minimum 0
```
So, you could make the two inputs related to water temp be autosizable.  But maybe this is too confusing and we should just have the warning?  Would like some other opinions.

**Comment by Myoldmopar** [comment]
Closed via #5970
